swift/ql/lib/codeql/swift/frameworks/Alamofire/Alamofire.qll

```diff
@@ -0,0 +1,39 @@
+/**
+ * Models for the Alamofire networking library.
+ */
+
+import swift
+private import codeql.swift.dataflow.DataFlow
+private import codeql.swift.dataflow.ExternalFlow
+private import codeql.swift.dataflow.FlowSources
+
+/**
+ * An Alamofire response handler type.
+ */
+private class AlamofireResponseType extends NominalTypeDecl {
+  AlamofireResponseType() {
+    this.getFullName() = ["DataResponse", "DownloadResponse"] or
+    this.getABaseTypeDecl() instanceof AlamofireResponseType
```

Comment:
Would this be equivalent to the following

```suggestion
    this.getABaseTypeDecl*().getFullName() = ["DataResponse", "DownloadResponse"]
```
? If so, I think that's more clear.

Reply:
I agree.